.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ on:
 env:
   CI: true
   UV_SYSTEM_PYTHON: 1
+  COVERAGE_CORE: sysmon
```

Comment:
Tested locally on Python 3.13 (this works on 3.12+). The time of execution of:
- `pytest`: ~5 seconds
- `pytest --cov` without this change: ~13 seconds
- `pytest --cov` with this change: ~5 seconds

Basically, this removes the overhead of coverage. It requires setting `branch = false` in `[tool.coverage.run]`.